src/software/Localization/main_SfM_Localization.cpp

```diff
@@ -93,7 +93,7 @@ int main(int argc, char **argv)
     << "  contains a single intrinsics and, if so, take this value as intrinsics for the query images.\n"
     << "  (OFF by default)\n"
     << "[-e|--export_structure] (switch) when switched on, the program will also export structure to output sfm_data.\n"
```

Comment:
As this process only provides a way to localize new images (without extending the current model) the only new information is the pose of the new cameras. If our goal is only to obtain those poses there is no need to export the structure again. This is just to limit the export data size if the models are big and have a lot of landmarks.